Documentation/ceph-advanced-configuration.md

```diff
@@ -590,3 +591,42 @@ ceph osd pool get replicapool crush_rule
 If the cluster's health was `HEALTH_OK` when we performed this change, immediately, the new rule is applied to the cluster transparently without service disruption.
 
 Exactly the same approach can be used to change from `host` back to `osd`.
+
+## Auto Expansion of OSDs
+### To scale OSDs Vertically
+
+If you need to auto grow the size of OSDs on a PVC based rook-ceph cluster whenever the OSDs have reached the storage threshold you can use Prometheus for it.
+
+You need to just run the script `increase-size-of-osd.sh`.
```

Comment:
This can be part of the previous paragraph:

> If you need to auto-grow the OSDs on a PVC-based Rook-Ceph cluster whenever the OSDs have reached the storage near-full threshold, you can use Prometheus. Prometheus can call the script `add-osd-capacity.sh` to do so.